src/cmd/run.js

```diff
@@ -181,23 +183,19 @@ export default async function run(
   const requiresRemote = !preInstall;
   let installed = false;
 
-  let runner;
-  let profile;
-  let client;
-  let runningFirefox;
-  let addonId;
-
-  let manifestData = await getValidatedManifest(sourceDir);
-
-  runner = new ExtensionRunner({
+  const runner = new ExtensionRunner({
```

Comment:
Hi @pragyes31 I took a brief look at this and it looks like the changes on `src/cmd/run.js` have a number of issues, even besides the issues that flow has detected on Travis (https://travis-ci.org/mozilla/web-ext/jobs/183711790#L692).

I would suggest to split this change in 2 or 3 commits to make the review easier (and, as a side effect, it would make easier also a rebase of the branch if needed):

- 1 commit related to the change to the eslintrc file
- 1 commit with the fixes that eslint is able to apply by running it on the project with the `--fix` command line option
- 1 additional commit related to the linting errors that cannot be fixed by eslint itself and have to be fixed manually (because they are probably the one that needs a more detailed review)